data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/DirectActionsWML/kill_fires_events.cfg

```diff
@@ -0,0 +1,41 @@
+#####
+# API(s) being tested: [kill]fire_event=yes
+##
+# Actions:
+# Put Alice on recall, kill Bob and Alice.

```

Comment:
Might be better to be a little more verbose here.

Reply:
I don't know how to get more detailed from what i added, most notes in tests are usually quite brief.

Reply:
I preferred Pentarctagon's suggestion from the old PR of "Put alice to the recall list." instead of "Put Alice on recall". It's small, but the extra words make it easier to read.